baybe/simulation/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""Functionality to "simulate" Bayesian DOE given a lookup mechanism.
+
+The term "simulation" can have two slightly different interpretations, depending on the
+applied context:
+
+*   It can refer to "backtesting" a particular DOE strategy on a fixed (finite)
+    dataset. In this context, "simulation" means investigating what experimental
+    trajectory we would have observed had we applied the recommender in a certain
+    defined context and restricted the possible parameter configurations to those
+    contained in the dataset.
+
+*   It can refer to the simulation of an actual DOE loop (i.e. recommending experiments
+    and retrieving the corresponding measurements) where the loop closure is realized
+    in the form of a callable (black-box) function that can be queried during the
+    optimization to provide target values.
+"""
+
+from baybe.simulation.core import simulate_experiment
+from baybe.simulation.transfer_learning import simulate_transfer_learning
+from baybe.simulation.wrappers import simulate_scenarios
```

Comment:
I am not happy with naming this `wrapper` - isn't all of this functionality somehow just providing a wrapper functionality? I'd actually prefer `scenarios`

Reply:
we have a separate open thread for this below, so I'll resolve this one